README.md

```diff
@@ -34,6 +36,13 @@ To receive notifications about new EKS-D releases, subscribe to the EKS-D update
 Source: https://github.com/cncf/artwork/tree/master/projects/kubernetes/certified-kubernetes
```

Comment:
It won't let me leave a comment above here, but just above here, you'll want to change the conformance badge to be for the 1.26 conformance.

Steps:
1. Delete the image file at `docs/contents/certified-kubernetes-1.25-color.svg`
1. Download https://github.com/cncf/artwork/blob/master/projects/kubernetes/certified-kubernetes/1.26/color/certified-kubernetes-1.26-color.svg
1. Save the downloaded image to `docs/contents/certified-kubernetes-1.26-color.svg`
1. On line 34 of this file...
    1. Change the image name to be for 1.26 (i.e. make it `docs/contents/certified-kubernetes-1.26-color.svg`)
    1. Replace the PR link (`https://github.com/cncf/k8s-conformance/pull/2452`) with your conformance PR, which is `https://github.com/cncf/k8s-conformance/pull/2507`
1. Update the runbook to notes these steps. It's fairly common for them not to have the badge done by our release, so also include a note that, if the new version's conformance badge isn't at `https://github.com/cncf/artwork/blob/master/projects/kubernetes/certified-kubernetes/1.<VERSION>/color/certified-kubernetes-1.<VERSION>-color.svg` (replace `<VERSION>` with the minor release, like `27`), do not worry about updating it for the initial docs release. This can be done whenever the badge is done at some point in the future.